packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/configmid.md

```diff
@@ -10,37 +12,3 @@ Create the Okta configuration file and add the keys and values for your applicat
    | `logoutRedirectUri` | The post-logout redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/` |
```

Comment:
Easier to give it the name in the Admin Console?

```suggestion
   | `logoutRedirectUri` | The sign-out redirect URI from the app integration that you created, such as `com.okta.dev-1234567:/` |
```

Reply:
Done. In the next commit.